ergo/foretold.py

```diff
@@ -0,0 +1,70 @@
+from matplotlib import pyplot as plt
+import torch
+import numpy as np
+import requests
+from ergo.ppl import uniform
+
+
+class ForetoldDistribution:
+    """"Aggregated distribution from a foretold question"""
+
+    def __init__(self, id):
+        """
+            id: measurableId, the second id in the URL for a foretold question
+        """
+        self.id = id
+        self.floatCdf = None
+        # previousAggregate is the most recent aggregated distribution
+        response = requests.post(
```

Comment:
small thoughts:

1. this indentation is a bit weird/confusing to me, I'd indent more
2. I'd do the response/j bit as a method that gets called, to keep the init really clean + simple
3. I'm not a fan of 1-character variable names like j, I'd prob call it json, or go ahead and pull out the data or something

don't feel strongly, would actually be curious if you think either of those suggestions would in fact make the style worse

Reply:
json is the name of a python module, so probably confusing to use as a variable name, but changed to something else

Reply:
I actually had an off-by-one error in the whole thing. Now it makes more sense, because the first ys should be 0, so we never sample a lower value than 0.